src/EditorFeatures/Test/Utilities/PatternMatcherTests.cs

```diff
@@ -528,9 +512,10 @@ private static IEnumerable<PatternMatch> TryMatchMultiWordPattern(string candida
         {
             MarkupTestFile.GetSpans(candidate, out candidate, out ImmutableArray<TextSpan> expectedSpans);
 
-            var matches = new PatternMatcher(pattern).GetMatches(candidate, includeMatchSpans: true);
+            var matches = ArrayBuilder<PatternMatch>.GetInstance();
```

Comment:
Oh, never mind, this is just the scratch array.  We are testing the 'free' codepaths for the pattern matcher itself